.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,144 @@
+name: Integration tests
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+    branches:
+      - main
+      - stable-*
+
+jobs:
+  splitter:
+    env:
+      source_dir: "./source"
+    runs-on: ubuntu-latest
+    outputs:
+      test_targets: ${{ steps.splitter.outputs.test_targets }}
+      test_targets_json: ${{ steps.splitter.outputs.test_targets_json }}
+      test_jobs: ${{ steps.splitter.outputs.test_jobs }}
+    steps:
+      - name: Checkout the collection repository
+        uses: actions/checkout@v3
+        with:
+          path: ${{ env.source_dir }}
+          fetch-depth: "0"
+
+      - name: list changes for pull request
+        id: splitter
+        uses: ansible-network/github_actions/.github/actions/ansible_test_splitter@main
+        with:
+          collections_to_test: ${{ env.source_dir }}
+          total_jobs: 8
+
+      - name: Display splitter output
+        run: |
+          echo "test_targets=${{ steps.splitter.outputs.test_targets }}"
+          echo "test_targets_json=${{ steps.splitter.outputs.test_targets_json }}"
+          echo "test_jobs=${{ steps.splitter.outputs.test_jobs }}"
+        shell: bash
+  integration:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    needs:
+      - splitter
+    if: ${{ needs.splitter.outputs.test_targets != '' }}
+    env:
+      source: "./source"
+      cloud_common: "./cloudcommon"
+      ansible_posix: "./ansible_posix"
```

Comment:
it should be nice to define all dependent collections into a `tests/integration/requirements.yml` file as we did for others collections

Reply:
I can do it as part of a separate PR. This PR is to fix the sanity issues.